src/libnyoci/nyoci-logging.h

```diff
@@ -34,40 +34,46 @@
 
 #if !VERBOSE_DEBUG
 
-#define CSTR(x)     (x)
+	#define CSTR(x)     (x)
```

Comment:
I'd rather significant changes in the whitespace not be combined with material additions or changes to a file, unless it is unavoidable. It makes the diffs harder to read.

That aside, I'm having trouble getting my brain to not be weirded out by the use of full indents, though. That's largely why I used single-space indents in `assert-macros.h`: indented preprocessor macros look weird to me after 20+ years of not indenting them at all. I found the single-space indentation improved readability for `assert-macros.h` without looking *too* weird to my eyes, but maybe I'm just being stubborn and tabs are really the way to go.